beetsplug/info.py

```diff
@@ -52,7 +53,11 @@ def emitter():
             tags[field] = getattr(mf, field)
         tags['art'] = mf.art is not None
         tags['path'] = displayable_path(path)
+        # create a temporary Item to take advantage of __format__
+        tags['item'] = Item(db=None, **tags)
```

Comment:
Hmm, good point. Maybe the thing to do would be to have both emitters just produce `Item`s *instead* of dictionaries. Then, the *later* machinery can be responsible for flattening the `Item` for display. That would certainly be simpler—but perhaps I'm missing something that would prevent it from working.